src/python/tests/appengine/handlers/cron/oss_fuzz_generate_certs_test.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""oss_fuzz_apply_ccs tests."""
+import sys
+import unittest
+
+from google.cloud import ndb
+import webapp2
+import webtest
+
+from datastore import data_types
+from handlers.cron import oss_fuzz_generate_certs
+from tests.test_libs import helpers as test_helpers
+from tests.test_libs import test_utils
+
+
+@unittest.skipIf(sys.version_info.major != 3, 'Python 3 only')
```

Comment:
there was some helper decorator for this ?

Reply:
We have python2_only but not a python 3 equivalent. It might make sense to for consistency, but hopefully both will be irrelevant soon. It would make cleanup after the migration is complete a little easier to have our own decorators in both cases, though.

Reply:
Added a python3_only decorator.